tests/framework/jailer.py

```diff
@@ -162,6 +162,27 @@ def jailed_path(self, file_path, create=False, create_jail=False):
             utils.run_cmd(cmd)
         return jailed_p
 
+    def copy_into_root(self, file_path, create_jail=False):
+        """Copy a file in the jail root, owned by uid:gid.
+
+        Copies a file in the jail root, creating the folder path if
+        not existent and changes their owner to uid:gid
```

Comment:
nit: use active voice, same as above.
```suggestion
        Copy a file in the jail root, creating the folder path if
        not existent, then change their owner to uid:gid.
```

Reply:
done